packages/web3torrent/package.json

```diff
@@ -10,6 +10,7 @@
   ],
   "dependencies": {
     "@rimble/connection-banner": "1.2.3",
+    "@sentry/browser": "^5.15.5",
```

Comment:
Let's pin this to an exact version and use the same version for rps so that only one version of the package is installed in the monorepo.

Reply:
done.

Reply:
Do you have a commit where this is fixed? I caret in front of the version number means that the version is not fixed.